Modules/Filtering/Denoising/test/CMakeLists.txt

```diff
@@ -40,6 +46,7 @@ itk_add_test(NAME itkPatchBasedDenoisingImageFilterTestRician
       2 1 12.283962106667094 2 2 200 0 1 RICIAN 0.1)
 itk_add_test(NAME itkPatchBasedDenoisingImageFilterTestPoisson
       COMMAND ITKDenoisingTestDriver
+    --compareRadiusTolerance 2
```

Comment:
FYI
_itkPatchBasedDenoisingImageFilterTestPoisson_ doesn't fail for me with min.
--compareRadiusTolerance 3
--compareIntensityTolerance 8

Reply:
Thanks for testing and feedback @issakomi . I'll change it to these values.